packages/hashi/src/kolibri.js

```diff
@@ -94,6 +94,7 @@ export default class Kolibri extends BaseShim {
        * @param {string[]} options.ids - an array of ids to filter by
        * @param {number} [options.page=1] - which page to return from the result set
        * @param {number} [options.pageSize=50] - the page size for pagination
+       * @param {Object} options.extra - extra parameters to pass to the fetchCollection call
```

Comment:
Likewise here.

Reply:
Yes I agree, all should be explicit and documented. No "extra" parameters.